cmd/cdi-apiserver/apiserver.go

```diff
@@ -113,6 +125,23 @@ func main() {
 		klog.Fatalf("Unable to get kube client: %v\n", errors.WithStack(err))
 	}
 
+	cache, err := cache.New(cfg, cache.Options{})
+	if err != nil {
+		klog.Fatalf("Unable to get cache: %v\n", errors.WithStack(err))
+	}
+
+	go func() {
+		if err := cache.Start(context.TODO()); err != nil {
+			klog.Infof("Unable to start cache: %v\n", errors.WithStack(err))
+		}
+	}()
+	// FIXME: WaitForCacheSync?
```

Comment:
You should be able to sync with something like
```go
informers := cache.Informers
if !informers.WaitForCacheSync(context.TODO()) {
  //Unable to sync
  klog.Fatalf("Unable to sync cache")
}
```

Reply:
I think you can just init informers for pv/sc
https://github.com/kubevirt/containerized-data-importer/blob/bf97a289057f9b699db42f1e0dac270c148af55a/pkg/apiserver/auth-config.go#L83-L93